gradle/dependencies.gradle

```diff
@@ -34,7 +34,7 @@ def build = [
     checkerDataflow         : ["org.checkerframework:dataflow:${versions.checkerFramework}",
                                "org.checkerframework:javacutil:${versions.checkerFrameworkNAFork}"],
     gradleErrorPronePlugin  : "net.ltgt.gradle:gradle-errorprone-plugin:0.0.8",
-    guava                   : "com.google.guava:guava:21.0",
+    guava                   : "com.google.guava:guava:22.0",
```

Comment:
make it `24.0-jre`?

Reply:
I'm trying to minimize potential incompatibility with other annotation processors.  If we decide to shade Guava I'll bump higher